lib/ts/recipe/emailpassword/types.ts

```diff
@@ -200,7 +200,17 @@ export type RecipeInterface = {
     resetPasswordUsingToken(input: {
         token: string;
         newPassword: string;
-    }): Promise<{ status: "OK" | "RESET_PASSWORD_INVALID_TOKEN_ERROR" }>;
+    }): Promise<
```

Comment:
Need to also change `passwordResetPOST`?

Reply:
If I remember correctly when we discussed this we did not want to add the `userId` to the FDI.

Reply:
Yes. We don't want to add it to the FDI. But we still want to add it as a result of this function since it can be useful for overrides. But we won't send it to the frontend (hence no change to FDI)

Reply:
I have made this change already.